Antidote/CallsManager.m

```diff
@@ -421,7 +435,7 @@ - (void)switchViewControllerForCall:(OCTCall *)call
 {
     AALogVerbose(@"%@", call);
 
-    if ([self.currentCallViewController isKindOfClass:[ActiveCallViewController class]]) {
+    if ([self isCurrentViewControllerReusableForCurrentCall:call]) {
```

Comment:
Yea it is. That's because ActiveCallViewController's are the only reusable ones in this case if it's in either a paused or active state. At the time, I was wondering if I should of done the check in the "if" clause or move it to another method. I chose the latter.

Is there a better naming I could of done?